emojivoto/emojivoto-web/webapp/README.md

```diff
@@ -0,0 +1,45 @@
+# Emojivoto webapp
+
+This app is written with React and bundled with webpack.
+
+## Local development
+
+Use the following to run the emojivoto go services and develop on the frontend.
+
+Start the emoji service
+```
+cd ../../emojivoto-emoji-svc
+export GRPC_PORT=8082
+go run cmd/server.go
+```
+
+Start the voting service
+```
+cd emojivoto-voting-svc
```

Comment:
Is this expected to be a continuation of the previous command block? If so, then I think it should be `cd ../emojivoto-voting-svc`.

Reply:
I was intending on opening them in different terminal windows (I could change that though!). I should clarify

Reply:
Different windows works for me! And I think if you move this to the main readme, then you can specify all paths from the root, which will make it clearer.

Reply:
yeah!!!